ci_build/azure_pipelines/compliance.yml

```diff
@@ -0,0 +1,50 @@
+jobs:
+
+- job: compliance
+  pool:
+    vmImage: "windows-2019"
+  steps:
+  - task: UsePythonVersion@0
+    displayName: 'Use Python 3.8.x'
+    inputs:
+      versionSpec: 3.8.x
+
+  - powershell: |    
+     python -m pip --disable-pip-version-check install -r $(Build.SourcesDirectory)\mobile\examples\speech_recognition\model\requirements.txt

```

Comment:
Do we need to install pytorch? How about use `onnxruntime-inference-examples/mobile/examples/basic_usage/model/requirements.txt`?

Reply:
I don't know. Probably it doesn't matter much. We can revisit later. This pipeline is just for static code analysis.